Docker-compose re-spin post (#260)
flowforge/website: src/blog/2022/11/respin-docker-compose-01.md

---
title: "Re-spin of Docker-Compose install package"
subtitle: Adding ability to locally build file-server container

> description: Details about patch release

Reviewer: ```suggestion
description: Details FlowForge v1.1.1
```

date: 2022-11-25
authors: ["ben-hardill"]
---

> After yesterdays 1.1.0 FlowForge release we noticed a few minor issues with the docker-compose install instructions.

Reviewer: ```suggestion
After [yesterdays 1.1.0 FlowForge release](https://flowforge.com/blog/2022/11/flowforge-1-1-released/) we noticed a few minor issues with the docker-compose install instructions.
```

> We had relied on the fact we are now publishing container images to Docker Hub to install the new `flowforge/file-server` container. At this time we are only building images for amd64 and arm64 and we had only tagged them with the current release number.

Reviewer: ```suggestion
We had relied on the fact we are now publishing container images to Docker Hub to install the new `flowforge/file-server` container. At this time we are only building images for amd64 and armv6. Further, it was only tagged them with the current release number.
```

Reviewer: My eyes lied to me. I read `amd64` twice. Sorry

Reviewer: ```suggestion
We had relied on the fact we are now publishing container images to Docker Hub to install the new `flowforge/file-server` container. At this time we are only building images for amd64 and arm64. Further, it was only tagged them with the current release number.
```

> This means that 1.1.0 release of Docker Compose could not be installed on other architectures e.g. armv7.

Reviewer: This is the last line of the previous paragraph. No need for the extra CR's :)
